src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -19,11 +19,16 @@ namespace Microsoft.ML
     public static class ConversionsExtensionsCatalog
     {
         /// <summary>
-        /// Hashes the values in the input column.
+        /// Create a <see cref="HashingEstimator"/>, which hashes the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
         /// <param name="catalog">The conversion transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of <see cref="System.UInt32"/>, or a <see cref="System.UInt32"/> based on whether the input column data types
```

Comment:
><see cref="System.UInt32"/>, or a <see cref="System.UInt32"/> [](start = 56, length = 61)

Can you see difference between `<see cref="System.UInt32"/>` and `<see cref="System.UInt32"/>` ? Because I don't.

Reply:
awesome :)

---
In reply to: [277079450](https://github.com/dotnet/machinelearning/pull/3394#discussion_r277079450) [](ancestors = 277079450)

Reply:
actually that is correct: a vector of ints or an int. 

I can add scalar. 

---
In reply to: [277099542](https://github.com/dotnet/machinelearning/pull/3394#discussion_r277099542) [](ancestors = 277099542,277079450)